sanitizers/os.go

```diff
@@ -115,7 +115,7 @@ func OsReadDir(hookId int, name string) ([]os.DirEntry, error) {
 	return os.ReadDir(name)
 }
 
-func OsReadLink(hookId int, name string) (string, error) {
+func OsReadlink(hookId int, name string) (string, error) {
```

Comment:
Weird decision by go folks to use lower-case naming for `link` here but good catch!